packages/cli/src/commands/verify/util.ts

```diff
@@ -0,0 +1,8 @@
+export const promiseResult = async <T>(promise: Promise<T>): Promise<[undefined, T] | [unknown, undefined]> => {
```

Comment:
what is the meaning of the function, why is this wrapper needed?

Reply:
a verify function calls a promise, we'd like to be aware immediately once the promise is resolved or rejected without it throwing. thus wrapping the promise where it returns the error instead of throwing it.